include/logger/loggerConfig.h

```diff
@@ -272,59 +276,104 @@ typedef struct _PWMConfig {
 #define DEFAULT_PWM_CONFIG {DEFAULT_PWM_CHANNEL_CONFIG, MODE_PWM_FREQUENCY, MODE_LOGGING_PWM_DUTY, DEFAULT_PWM_DUTY_CYCLE, DEFAULT_PWM_PERIOD}
 
 
-#define OBD2_CHANNELS 20
+#define DEFAULT_CAN_BAUD_RATE 500000
+
+typedef struct _CANConfig {
+    unsigned char enabled;
```

Comment:
Changing to boolean. research shows that it defaults to 8 bits, which is what I originally wanted.